maskrcnn_benchmark/data/README.md

```diff
@@ -70,19 +70,22 @@ ln -s /path/to/cityscapes datasets/data/cityscapes
 1. Download gtFine_trainvaltest.zip from https://www.cityscapes-dataset.com/downloads/ (login required)
 2. Extract it to /path/to/gtFine_trainvaltest
 ```
-gtFine_trainvaltest
-|_ gtFine
+cityscapes
+|_ gtFine_trainvaltest.zip
+|_ gtFine_trainvaltest
+   |_ gtFine
 ```
 3. Run the below commands to convert the annotations
+**NOTE: Detectron AND maskrcnn_benchmark is required**
```

Comment:
This sounds good to me, but from a glance at the implementation, it is really easy to get rid of the dependency on Detectron.

Reply:
Lazily I have copied `boxes.py` and `segms.py` from detectron to the maskrcnn utils, but then I saw that in boxes there was another level of dependency - and I got like "meh... just quickly install detectron", but then I had to resolve the caffe2 stuff :D so yeah... what would you suggest?

I can trace all the functions back in the detectron lib and copy them together to the maskrcnn lib or follow along [#447 ](https://github.com/facebookresearch/maskrcnn-benchmark/issues/447#issuecomment-464658449)

Reply:
I think you should follow what I've mentioned in #447. There is really only one function that needs to be ported from Detectron, the other is already in this repo. Let me know if you have further questions!